controllers/controller.js

```diff
@@ -86,7 +86,9 @@ exports.sendComment = (req, res, next) => {
 exports.removeComment = (req, res, next) => {
 	const commentId = req.params.comment_id;
 
-	deleteComment(commentId).then(() => {
+	Promise.all([deleteComment(commentId), doesResourceExist('comments', 'comment_id', commentId)]).then(() => {
```

Comment:
For this one you don't _**need**_ to check if the comment exists in this way - if the `rows` property on the results object is an empty array, then it means nothing was deleted - just a way to reduce the amount of queries to the database where possible